website/cue/reference/components/sinks/gcp_stackdriver_metrics.cue

```diff
@@ -106,6 +106,13 @@ components: sinks: gcp_stackdriver_metrics: {
 				syntax:  "literal"
 			}
 		}
+		max_batch_size: {
+			common:      false
+			description: "Maximum time series batch size used when sending metrics to GCP."
+			required:    false
+			warnings: []
+			type: uint64|null
```

Comment:
For the `type` here, you need to provide an object. Here's an example:

```cue
type: uint: {
    default: 200
    unit: "bytes"
}
```

Reply:
Thanks! It worked.